solution/ui/regulations/eregs-vite/src/components/resources/ResourceExportBtn.vue

```diff
@@ -93,7 +93,9 @@ export default {
                     page,
                     partDict: this.partDict,
                     categories: this.categories,
-                    q: this.query
+                    q: this.query,
+                    partDict:"all",
```

Comment:
there are two `partDict` keys, with the second one (`"all"`) overwriting the first one (`this.partDict`).

Also regarding formatting: there should be a space between the key and the val

`partDict: "all",`